RELEASES.md

```diff
@@ -77,6 +77,24 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
         
     c. Create a PR on github into the release branch. Ask in the UA channel on mattermost for review.
 
+    d. When reviewing the release PR, please use the following guidelines when reviewing the new changelog entry:
+
+       * Is the version correctly updated ? We must ensure that the new version on the changelog is
+         correct and it also targets the latest Ubuntu release at the moment.
+       * Is the entry useful for the user ? The changelog entries should be user focused, meaning
+         that we should only add entries that we think users will care about (i.e. we don't need
+         entries when fixing a test, as this doesn't provide meaningful information to the user)
+       * Is this entry redundant ? Sometimes we may have changes that affect separate modules of the
+         code. We should have an entry only for the module that was most affected by it
+       * Is the changelog entry unique ? We need to verify if the changelog entry is not already
+         reflected n an early version of the changelog. If it is, we need not only to remove but double
```

Comment:
missing `i` here I think (:

Reply:
or `o`

Reply:
My english skills cannot differentiate `in` the changelog from `on` the changelog. I suggest asking a native speaker hahahaha

Reply:
Haha! I think `in` works better here :)
And a couple minor nits: I'd suggest "We need to verify ~~if~~ _that_ the changelog entry is not already reflected _in_ an ~~early~~ _earlier_ version of the changelog".